### ISSUE TITLE
Quest  [Raising Hodir's Spear] not working as inteded.

### ISSUE THREAD

**Issue by Samfuzi** [dalaranwow/dalaran-wow#4451]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
This quest is not possible to complete. The target item "Everfrost Shard" is not available for looting.
**Expected behaviour**: Tell us what should happen instead
The item "Everfrost Shard" should spawn in the inner chamber of Hibernal Cavern in Storm Peaks.
**Steps to reproduce the problem**:

1. Abandon quest  [Raising Hodir's Spear]
2. Accept quest  [Raising Hodir's Spear]
3. Item still not available for looting. It just don`t seem to exist on the server.

**Include proofs for this behaviour**
![no everfrost shards to loot](https://cloud.githubusercontent.com/assets/25851043/23078062/6f805d92-f547-11e6-8a8e-108e6e13f405.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/
More info from wowhead:
http://www.wowhead.com/item=42541/everfrost-shard#objective-of

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Fixed update pending on PTR https://github.com/dalaranwow/dalaran-wow/issues/3988
